### PR TITLE
Scripts for handing E2E testing in test-infra

### PR DESCRIPTION
**Pull request by schylek**

### CODE REVIEW

File: vertical-pod-autoscaler/hack/deploy-for-e2e.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+COMPONENTS="recommender updater admission-controller"
+
+if [ $# -ne 0 ]; then
+  COMPONENTS=$*
+fi
+
+for i in ${COMPONENTS}; do
+  (
+    cd ${SCRIPT_ROOT}/${i} && make release
```
Comment (tkulczynski) [inline vertical-pod-autoscaler/hack/deploy-for-e2e.sh]: Hint:
```
make --directory ${i} release
```
It's a nice shortcut that also goes back to your previous directory afterwards.

File: vertical-pod-autoscaler/hack/vpa-process-yamls.sh [hunk 2]
```diff
@@ -14,9 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -o errexit
```
Comment (tkulczynski) [inline vertical-pod-autoscaler/hack/vpa-process-yamls.sh]: The [no]errexit strategy is confusing to me, especially in nested scripts calling scripts. I would be glad to have it set always and instead use "|| true" on the commands that we actually expect to fail and are ok with that.

Comment (schylek) [reply vertical-pod-autoscaler/hack/vpa-process-yamls.sh]: Redinements applied.

**Comment by tkulczynski** [comment]
Thanks, LGTM now.

**Pull request merged**